bb.edn

```diff
@@ -1,4 +1,5 @@
 {:paths ["scripts"
+         ;; ensure lint checks integration tests:
```

Comment:
Not only liniting, but clojure-lsp as a whole, like when checking unused-public-var which will only work properly if those paths are on classpath for clojure-lsp during initialize

Reply:
I've reverted all the changes to `bb.edn`, so this is ready for another review.